### ISSUE TITLE
Rework of the documentation

### ISSUE THREAD

**Issue by LucileDT** [Serhioromano/bootstrap-calendar#721]
Hello!

I'm trying to install the calendar on a project. I think I'll succeed ! If I achieve it, would it be possible for me to edit your documentation?

I would like to:
- update the README with a more user friendly installation tutorial
- update the list of common issues in the wiki https://github.com/Serhioromano/bootstrap-calendar/wiki/Common-errors to make it more readable and user friendly
- close issues about solved problems/questions asked a long time ago
- set up labels on issues depending on:
  - it's a user side problem
  - it's a calendar bug
  - it's a demand of improvement

I don't know what you think about it, but it seems that you devs are pretty bothered by community asking every time the same questions and "whining" about how "bad" the documentation is. Maybe I can help you with it, if you want to :slightly_smiling_face:

**Comment by LucileDT** [comment]
I've already begun to work:

## README.md
- [ ] Update screenshot (pull request pending)
- [ ] Update way to intall to a more user friendly thing (be done when wiki tasks will be finished)

## Wiki
- [ ] Home
    - [ ] update screenshot
    - [ ] update links to other pages
        - [ ] Introduction
        - [ ] Getting started
        - [ ] How to use
        - [ ] API
        - [ ] Examples
        - [ ] Customize Weektime
- [ ] Introduction
    - [ ] determine what devs want to put in it
- [ ] Getting started
    - [ ] list of external dependencies
    - [ ] download
        - [ ] with bower
        - [ ] with composer
        - [ ] with git
        - [ ] manually
    - [ ] usage
        - [ ] file tree and internal dependencies
        - [ ] minimal setup to display default calendar
- [ ] How to use
    - [ ] describe complex usage of the calendar
- [ ] Options
    - [ ] rework to set tables instead of lists
    - [ ] add events class options
- [ ] Examples
    - [ ] create page
    - [ ] setup some examples 
        - [ ] localized calendar
        - [ ] events opening in modals
        - [ ] classes of events
        - [ ] disable/enable views
        - [ ] interaction with buttons

**Comment by kirkita** [comment]
Agree, plugin is the best, but documentation is poor :(

**Comment by LucileDT** [comment]
@kirkita hello! What do you need first? I'll update the documentation for your needs if I can.

Furthermore, maybe I can help you: I managed to install the calendar and customize it in various ways, so, even if I didn't take the time already to write the documentation, I know some useful tricks.

**Comment by erniomaldo** [comment]
_(Caution: translated with translate tools from google)_
Hi there @LucileDT, thanks for your work, i suggest the following list in order from most important to least important:

- add new classes of events (default is only 6 diff classes)
- setup the start and end of day view (default is 06:00 to 22:00)
- interaction with buttons
- how to tire a function on click event (instead url or modal)
- classes of events

Items not included in the list are not priorities from my point of view, since I am new to web development.

**Comment by LucileDT** [comment]
@erniomaldo hello!
I think there is an example in the project, did you try to open it? I just came back from holidays, I will try to do what you asked, will you be here if I need more details for the points you need?

**Comment by erniomaldo** [comment]
Yes, the example code of implementation is named app.js, but its not complete at all and there is no more info, if you need help with somenthing let me know it ;)

**Comment by kirkita** [comment]
@LucileDT hey, Thank you. I was so buisy not viewed your response. I already make my calendar to work. Thank you again ;)
